src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
+  const arr = [];
```

Comment:
```suggestion
  const arrayOfStates = [];
```
Let's give more meaningful names to the variables